src/docfx/lib/git/GitCommitProvider.cs

```diff
@@ -61,13 +61,16 @@ public static async Task<GitCommitProvider> Create(string repoPath, string cache
             return new GitCommitProvider(repoPath, cacheFilePath, await LoadCommitCache(cacheFilePath));
         }
 
-        public List<GitCommit> GetCommitHistory(string file)
+        public List<GitCommit> GetCommitHistory(string file, string branch = null)
```

Comment:
default branch as `string.Empty`?

Reply:
I believe both are OK, even the default branch is string.empty, user still gets chance to pass branch 'null'

---
In reply to: [229918800](https://github.com/dotnet/docfx/pull/3546#discussion_r229918800) [](ancestors = 229918800)